test/extensions/filters/http/cache/http_cache_test.cc

```diff
@@ -374,6 +374,28 @@ TEST_F(LookupRequestTest, NotSatisfiableRange) {
   EXPECT_FALSE(lookup_response.has_trailers_);
 }
 
+TEST_P(LookupRequestTest, ResultWithBodyAndTrailersMatchesExpectation) {
+  request_headers_.setReferenceKey(Http::CustomHeaders::get().CacheControl,
+                                   GetParam().request_cache_control);
+  const SystemTime request_time = GetParam().request_time, response_date = GetParam().response_date;
+  const LookupRequest lookup_request(request_headers_, request_time, vary_allow_list_);
+  const Http::TestResponseHeaderMapImpl response_headers(
+      {{"cache-control", GetParam().response_cache_control},
+       {"date", formatter_.fromTime(response_date)}});
+  const uint64_t content_length = 5;
+  const LookupResult lookup_response =
+      makeLookupResult(lookup_request, response_headers, content_length, /*has_trailers=*/true);
+
+  EXPECT_EQ(GetParam().expected_cache_entry_status, lookup_response.cache_entry_status_);
+  ASSERT_TRUE(lookup_response.headers_);
+  EXPECT_THAT(*lookup_response.headers_, Http::IsSupersetOfHeaders(response_headers));
+  EXPECT_THAT(*lookup_response.headers_,
+              HeaderHasValueRef(Http::CustomHeaders::get().Age, GetParam().expected_age));
```

Comment:
i'm trying to follow this test flow and it's not obvious to me where the age gets populated. Can you add some comments to the code explaining this?

Reply:
I attempted to explain where it is set. Let me know if there is something specific I should mention.

Reply:
Got it, thanks.